apps/mark-scan/frontend/src/pages/admin_screen.tsx

```diff
@@ -184,6 +196,12 @@ export function AdminScreen({
         <P>
           <SignedHashValidationButton apiClient={apiClient} />
         </P>
+        <H6 as="h2">Diagnostics</H6>
+        <P>
+          <Button onPress={() => setIsDiagnosticsScreenOpen(true)}>
+            View Diagnostics
```

Comment:
I added `View` because all the other buttons were actions, but happy to remove it if it seems unneeded

Reply:
Either works for me

Reply:
<img width="419" alt="Screenshot 2024-09-20 at 9 02 25 AM" src="https://github.com/user-attachments/assets/22249955-ea63-4f23-b0f5-07b222c55213">

Renaming the title to "System" and the button to "Diagnostics", as this is more similar verbiage to what is used in the other apps. I figure we can change them all in the future if we  need, but better if we stay consistent for now.